### ISSUE TITLE
Odd "gibberish" in admin view

### ISSUE THREAD

**Issue by mwweb** [vdm-io/Joomla-Component-Builder#56]
I started looking through my views to ensure all is looking the way that I want, and I noticed in one of my admin views (single item view) that there are script tags, even though I have no custom scripts loading in that view, and I have no idea WHERE they are coming from.

Have you come across anything like below?

```
<script type="text/javascript">

	D“8ÀŽN

</script>
```

**Comment by Llewellynvdm** [comment]
I would say go to that view and check if you have that code in any of its fields. I have not seen that before.

**Comment by mwweb** [comment]
That was actually one of the first things I did. And even checked the record in the database, and saw nothing. I guess I'll have to work on rebuilding that view today, and see if it does it after a compile.

**Comment by Llewellynvdm** [comment]
Hmmm does it add that gibberish during compilation? So you see that in the edit view of the item?

**Comment by mwweb** [comment]
I'm seeing it in the code of the view. I actually came across this when adding some custom code, the code wasn't working, so I was digging through. I deleted the custom code, still there. I just created a new view, very similar to this one, and it's putting it there too. I just deleted 99% of the field I put in, and it's not there. So, it must be something in one of the fields being added. I'll keep this updated, as I add a few fields, compile, and when I start seeing it again, dig into those fields.
